docusaurus/docs-internals/editor-controls/data-controls/customCss.md

```diff
@@ -0,0 +1,212 @@
+---
+sidebar_position: 1
+toc_max_heading_level: 4
+---
+# Custom Css
+
+The custom css option allows you to style the component as you like by adding your own styles.
+
+Example of customCss control:
```

Comment:
remove custom css, your control is `codeMirror`, so: "Example of code mirror control:"